docs/linting/TROUBLESHOOTING.md

```diff
@@ -119,12 +119,12 @@ As of our v4.0.0 release, this also applies to types.
 If you use global types from a 3rd party package (i.e. anything from an `@types` package), then you will have to configure ESLint appropriately to define these global types.
 For example; the `JSX` namespace from `@types/react` is a global 3rd party type that you must define in your ESLint config.
 
-Note, that for a mixed project including JavaScript and TypeScript, the `no-undef` rule (like any role) can be turned off for TypeScript files alone by adding an `overrides` section to .eslintrc.json:
+Note, that for a mixed project including JavaScript and TypeScript, the `no-undef` rule (like any role) can be turned off for TypeScript files alone by adding an `overrides` section to `.eslintrc.cjs`:
```

Comment:
If this is `.cjs`, the block below should be updated as well. Also I suggest wrapping it with `module.exports = {...}` to avoid having fragments of, in fact, invalid syntax

Reply:
oops thanks!
I was just blindly updating extensions without paying too much attention